src/serializers.ts

```diff
@@ -100,6 +100,10 @@ const disablePaths = (paths?: string[]) => {
 
 const enablePaths = (paths?: string[]) => {
     forEach(serializers, (value, key) => {
+        if (paths && paths.includes(key)) {
+            delete serializers[key];
```

Comment:
This will delete the whole serialization, that can contain also other logic apart of disabling some fields. That is for me not a feature that should "enableFields" option provide.

For example: if serializer contains addition of some fields (logger name, ...), it will be deleted as well.